QWeb/internal/secrets.py

```diff
@@ -101,6 +109,34 @@ def _filtered_start_keyword(keyword: Keyword) -> None:
             LOGGER._other_loggers[0].log_message = lambda x: None
 
 
+def _filtered_start_library_keyword(data: Keyword, implementation: Keyword, result: Keyword):
```

Comment:
Add docstring.
Specify that this is function for RFW version 7+

Reply:
added and also end_library_keyword docstring modified